src/components/modal/Snackbar/style.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@emotion/react';
+
+import zIndexes from '@/styles/zIndexes';
+
+export const snackbarWrapperStyle = css`
+  position: fixed;
+  left: 50%;
+  bottom: 80px;
+  padding: 13px 30px;
+  min-width: 100px;
+  border-radius: 40px;
+  text-align: center;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.3);
+
+  transition: 0.6s;
+  transform: translate(-50%, 0);
```

Comment:
요런건 translateX 가 더 직관적이겠다는 생각이 드네요 ㅋㅋ;; 
```suggestion
  transform: translateX(-50%);
```

Reply:
`snackbarShowStyle` 혹은 `snackbarHideStyle` 에 의해서 덮어쓰기 되는 부분이라 해당 줄은 삭제했습니다.

다만 `snackbarShowStyle`의 `translate`의 경우에는 `snackbarHideStyle`에 의해 `-40px`된 `y`를 `0`으로 만들기 위한 목적이라 그대로 두었습니다!

관련 커밋 : [2a51f77](https://github.com/telbby/telbby-frontend/pull/28/commits/2a51f7736550cf6373702dc2b95649f0d8a253e5)

Reply:
아!! 그 생각을 못했네요 죄송합니다 😅